Source/Core/Core/HW/DSPHLE/UCodes/UCodes.cpp

```diff
@@ -293,6 +293,7 @@ std::unique_ptr<UCodeInterface> UCodeFactory(u32 crc, DSPHLE* dsphle, bool wii)
 
   case 0x008366af:
   case 0x078066ab:
+  case 0x5ad4d933:
```

Comment:
Just curious, is there a reason these aren't using constants?

Reply:
I don't think there's any reason not to other than that there aren't super obvious names for them (I've got constants in ASnd.cpp and AESnd.cpp, but AX.cpp/AXWii.cpp and Zelda.cpp don't have them and there isn't any easy way to get more information about them either).

Reply:
Ok, I didn't know if it'd be helpful to put the constant here instead.  But this function is mostly non-constants with comments.  So maybe just a comment instead?

Reply:
I decided to try out the constants, and it ended up being fairly easy to move these ones to the header file, so I've switched to using them. That should make it easier to find the more detailed comments associated with the constants. (I don't think it'd be useful to change the other ones, apart from maybe card and GBA - note that `UCODE_ROM` and `UCODE_INIT_AUDIO_SYSTEM` are magic values that _aren't_ actual hashes, which is kinda jank.)